source/Octopus.Manager.Tentacle/TentacleConfiguration/SetupWizard/TentacleSetupWizardModel.cs

```diff
@@ -519,9 +520,9 @@ public string SelectedSpace
                 selectedSpace = value;
                 OnPropertyChanged();
                 if (!string.IsNullOrEmpty(selectedSpace))
-#pragma warning disable 4014 // we want this to be async
-                    LoadSpaceData(async client => await LoadSpaceSpecificData(client));
-#pragma warning restore 4014
+                {
+                    var _ = LoadSpaceData(LoadSpaceSpecificData, CancellationToken.None);
+                }
```

Comment:
Unsure on how to best do this but just calling this and letting it run in the background seems like the accepted solution online.

Reply:
Hrm, this does seem a bit dodgy... I feel like this might have problems if there is an exception (or the exception might just get swallowed 🤔 )

Reply:
The actual work in LoadSpaceData is done in a `try...catch...finally` so I think it's ok in terms of exceptions...

Reply:
Also the way it was written before was effectively the same - it ran LoadSpaceData as an async method without awaiting the task.